packages/components/src/components/Task/Task.test.js

```diff
@@ -40,6 +40,34 @@ it('Task renders steps in expanded state', () => {
   expect(queryByText(/a step/i)).toBeTruthy();
 });
 
+it('Task renders first step in expanded Task with no error', () => {
+  const steps = [
+    { id: 'step', stepName: 'a step', reason: 'Completed' },
+    { id: 'step-two', stepName: 'a step two', reason: 'Completed' }
+  ];
+  const { queryByText } = renderWithIntl(
+    <Task {...props} expanded steps={steps} />
+  );
+  expect(
+    queryByText('a step').parentNode.parentNode.getAttribute('data-selected')
```

Comment:
can we achieve this without relying so much on the DOM structure?

maybe something like:
- query for element with attribute `data-selected`
- within that matched element, query by text for the step name